jingo/tests/django_app/helpers.py

```diff
@@ -0,0 +1,6 @@
+from jingo import register
```

Comment:
mind adding a comment to this file that it must not be explicitly imported anywhere in the project? that's what initially hid this bug